js/coinex.js

```diff
@@ -3460,23 +3460,40 @@ module.exports = class coinex extends Exchange {
     async transfer (code, amount, fromAccount, toAccount, params = {}) {
         await this.loadMarkets ();
         const [ marketType, query ] = this.handleMarketTypeAndParams ('transfer', undefined, params);
-        if (marketType !== 'spot') {
-            throw new BadRequest (this.id + ' transfer() requires defaultType to be spot');
-        }
         const currency = this.safeCurrencyCode (code);
         const amountToPrecision = this.currencyToPrecision (code, amount);
+        const marginAccountId = this.safeInteger (query, 'marginAccountId');
```

Comment:
Can we code it in a way that would allow us to specify the marginAccountId as one of fromAccount / toAccount in the args to avoid having an exchange-specific parameter here?

Reply:
Yeah I removed the exchange specific parameter, and just left it so the user has to specify the integer representing the account in fromAccount and toAccount